debian/patches/specify-max-resolution.patch

```diff
@@ -5,44 +5,109 @@ In the change introduced by https://codereview.chromium.org/872623002, Chromium
 There is a patch in libva (http://cgit.freedesktop.org/vaapi/intel-driver/commit/?id=9a20d6c34cb65e5b85dd16d6c8b3a215c5972b18) that asks what the maximum resolution supported by the hardware is. This change should be present in Xenial and newer, so rely on libva on returning a resolution. If it doesn't, then hardcode it to 1920x1088.
 
 
-Index: dev/media/gpu/vaapi_wrapper.cc
+Index: chromium-browser-60.0.3107.4/media/gpu/vaapi_wrapper.cc
 ===================================================================
---- dev.orig/media/gpu/vaapi_wrapper.cc
-+++ dev/media/gpu/vaapi_wrapper.cc
-@@ -460,7 +460,11 @@ bool VaapiWrapper::GetMaxResolution_Lock
+--- chromium-browser-60.0.3107.4.orig/media/gpu/vaapi_wrapper.cc
++++ chromium-browser-60.0.3107.4/media/gpu/vaapi_wrapper.cc
+@@ -7,6 +7,11 @@
+ #include <dlfcn.h>
+ #include <string.h>
+ 
++#include <string>
++#include <fstream>
++#include <sstream>
++#include <iostream>
++
+ #include "base/bind.h"
+ #include "base/callback_helpers.h"
+ #include "base/logging.h"
+@@ -442,6 +447,15 @@ bool VaapiWrapper::AreAttribsSupported_L
+   return true;
+ }
+ 
++#define VA_FB_RES_DEFAULT_WIDTH 1920
++#define VA_FB_RES_DEFAULT_HEIGHT 1088 // Yes, this should be 1088.
++int va_fb_res_width=VA_FB_RES_DEFAULT_WIDTH, va_fb_res_height=VA_FB_RES_DEFAULT_HEIGHT;
```

Comment:
Because this variable is being used only within the `VaapiWrapper` class (and its methods), you should be able to add a private `int` variable into the `VaapiWrapper` class definition (in`media/gpu/vaapi_wrapper.h`), and reduce the visibility of the variable. You would then modify the constructor of `VaapiWrapper` to set the variables to be the default width and height defines.